answers.txt

```diff
@@ -0,0 +1 @@
+﻿What is a gitignore file and what should it include for a Java project? 
Answer: gitignore file is a text file that tells Git which files or folders to ignore in a project. In Java, gitignore includes all the packages and temp files that need to be ignored.

What does fast-forward in Git mean? 
Answer: A fast-forward is what Git does when you merge or rebase against a branch that is simply ahead the one you have checked-out.

Why should you have different branches? 
Answer: Branching allows each developer to branch out from the original code base and isolate their work from others. It also helps Git to easily merge versions later.

In a project how many remote Git repositories do you usually have?
Answer: You can have as many remotes as you want, but you can only have one remote named "origin".

```

Comment:
Make sure to break a long line into smaller lines. Each line doesn't exceed 80 characters. Easier to read in the browser.